stark_node:
  ros__parameters:
    port_l: "/dev/ttyUSB0"
    port_r: "/dev/ttyUSB1"
    baudrate: 460800 # 二代灵巧手默认波特率
    slave_id_l: 0x7e # 二代灵巧手默认左手为0x7e, 右手为0x7f
    slave_id_r: 0x7f # 二代灵巧手默认左手为0x7e, 右手为0x7f
    firmware_type: 3  # V2tandard 二代灵巧手
    protocol_type: 1  # 1: Modbus RTU, 2: CANFD
    log_level: 1      # Info

    log_screen: false  # 是否显示终端日志